.github/workflows/ci.yml

```diff
@@ -36,10 +36,12 @@ jobs:
       - uses: actions/checkout@v3
       - name: Import database
         run: |
-          docker cp ./drupal/database.php.gz drupal:/opt/drupal/web
-          docker exec --workdir /opt/drupal/web drupal php ./core/scripts/db-tools.php import database.php.gz
+          docker cp ./drupal/settings.ci.php drupal:/opt/drupal/web/sites/default/settings.local.php
+          docker exec drupal ./import-db
       - name: Ensure Drupal is installed
-        run: curl --fail http://localhost/jsonapi
+        run: |
+          curl --fail http://localhost/jsonapi
+          docker exec drupal drush status
```

Comment:
This will help confirm that Drupal is actually installed and connected to the database that we think it is.